test-app/.prettierrc.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+
+module.exports = {
```

Comment:
Should it have settings for endOfFile and singleQuote like it did: https://github.com/adopted-ember-addons/ember-launch-darkly/blob/master/.prettierrc?